plugins/unreal/Open3DStream/Source/Open3DStream/Private/Open3DStreamSource.cpp

```diff
@@ -14,9 +14,11 @@ using namespace MyGame::Sample;
 // E:\Unreal\UE_4.25\Engine\Source\Runtime\LiveLinkInterface\Public\LiveLinkTypes.h
 // E:\Unreal\UE_4.25\Engine\Plugins\Runtime\AR\Apple\AppleARKit\Source\AppleARKitPoseTrackingLiveLink\Private\AppleARKitPoseTrackingLiveLinkSource.cpp
 
-FOpen3DStreamSource::FOpen3DStreamSource(const FText &InUrl, double InTimeOffset)
+FOpen3DStreamSource::FOpen3DStreamSource(const FText &InUrl, const FText &InKey, const FText &InProtocol, double InTimeOffset)
 	: bIsInitialized(false)
 	, Url(InUrl)
+	, Key(InKey)
```

Comment:
Add 'm' prefix

Reply:
Or stick with UE4 style?

Reply:
I know I'm a bit all over the place with the naming, but I'm no too worried about the formatting of the code right now as it's changing rapidly.  Members come and go on a daily basis.   I'd rather focus on getting a rock solid streaming working and general class layout and if we feel that is good we can reformat the names as needed.

Reply:
I don't have a strong opinion about what style to use - maybe as this is part of the unreal code it can follow their style, but the open3dstreamlib can have a different style if we want.